spec/registry.xml

```diff
@@ -8,6 +8,10 @@
   <validationRule id="timeZoneId" regex="[a-zA-Z/]+"/>
   <validationRule id="anythingNotEmpty" regex=".+"/>
   <validationRule id="iso8601" regex="\d{4}-\d{2}-\d{2}T\d{2}:\d{2}:\d{2}"/>
+  <validationRule id="xmlDate"
+    regex="-?([1-9][0-9]{3,}|0[0-9]{3})-(0[1-9]|1[0-2])-(0[1-9]|[12][0-9]|3[01])(Z|[+-]((0[0-9]|1[0-3]):[0-5][0-9]|14:00))?"/>
+  <validationRule id="xmlTime"
+    regex="(([01][0-9]|2[0-3])(:[0-5][0-9]){2}(\.[0-9]+)?|24:00:00(\.0+)?)(Z|[+-]((0[0-9]|1[0-3]):[0-5][0-9]|14:00))?"/>
```

Comment:
You're missing at least one production here, unless you think `iso8601` covers that? (using a different syntax??)

Is there some reason we can't reference XMLSchema directly?

Reply:
In case you're referring to the `:datetime` operand, changing its validation rule here would be scope creep.

To refer to external specs, we need language for how to do that, and that's not currently supported by `registry.dtd`.

Reply:
Agreed. `xsd` recommends itself, except I'm not sure how compatible the number types are with JSON number (in fact, I'm sure they aren't compatible)

I suspect we should make the registry's format completely unpromised in Tech Preview--even the use of XML. The normative bits are the default registry prose right now anyway. Let's solve this good and proper--and not over the weekend. I don't think anyone's implementation depends on the XML file now. We shouldn't encourage anyone to depend on it being any particular way in the future.

Thoughts?